CRM/Grant/Form/Task/Update.php

```diff
@@ -67,7 +67,7 @@ public function buildQuickForm() {
     $this->addElement('text', 'amount_granted', ts('Amount Granted'));
     $this->addRule('amount_granted', ts('Please enter a valid amount.'), 'money');
 
-    $this->addDate('decision_date', ts('Grant Decision'), FALSE, array('formatType' => 'custom'));
+    $this->add('datepicker', 'decision_date', ts('Decision Date'), [], FALSE, ['time' => FALSE]);
```

Comment:
We should use the same label on Add/Update/View grant as on this bulk update. So either change to Decision Date everywhere, or leave as Grant Decision here.

Reply:
Elsewhere it is "Grant Decision Date" so this change makes it a bit more standard without being too long to fit in the form. IMO "Grant Decision Date" is redundant on a grant form, so we could consider changing it elsewhere but I don't think that should block this PR.

Reply:
![2019-01-24_13-42-37](https://user-images.githubusercontent.com/300943/51700809-00e94880-1fde-11e9-80d9-0ebd8e3ae587.png)

Many of the fields on the Grant object have an extraneous 'Grant' in their name, and the name is Grant Decision everywhere else in the interface. Introducing inconsistent in the labels for a field is not a good idea. I'd prefer for you to drop that change. 

If you want to change that name to remove Grant from it, or to do that for some other ones like Grant Report Due or Grant Report Received that would be great in a separate MR that caught the label on all forms.

Reply:
Hmm. Looking around I see there is already inconsistency, with Decision Date being used on the criteria for the Find Grants page, which also has several other labels changed from how they appear on the View/Edit forms.

So I'll back off a bit and say I recommend weakly that you revert that label change.

Reply:
I've reverted it. Agree we should clean up labels in another PR.